### ISSUE TITLE
DNS not working out of the box after migrating to Valet+ and changing TLD

### ISSUE THREAD

**Issue by dvrgoc** [weprovide/valet-plus#269]
Hi guys, 

I may have run into an issue with your setup, so I would like to share with you what happened and my observations and conclusions.

I am running OSX 10.13.6 (High Sierra) and was using localhost setup as described in [3pt GetGrav article](https://getgrav.org/blog/macos-mojave-apache-ssl) for local projects. Important note: I was not using `dnsmasq` service (I was manually entering entries in /etc/hosts and httpd-vhosts.conf files).

Now that I have migrated to Valet+ and I have changed my TLD domain to "loc" in order to be backward compatible my old sites:
```bash
~ ᐅ valet domain
loc
```

I tried creating a new custom site in my parked folder: 
```bash
mkdir ~/valet/custom
cd ~/valet/custom
echo '<h1>hello valet plus</h1>' > index.php
```

## Expected result
`custom.loc` is accessible in browser

## Actual result
`custom.loc` is not accessible in browser, pinging domain in terminal fails

### Debugging
- If `127.0.0.1 custom.loc` entry is added to `/etc/hosts` file , the site is working (but this suggests I have to manually add any subsequent domain entry over and over again).
- In `/etc/resolver/` folder, i see two files:
  - loc
  - test
- In `~/.valet/dnsmasq.conf`, it still says `address=/.test/127.0.0.1`

### Solution
After making the following changes: 
- update `~/.valet/dnsmasq.conf`to  `address=/.loc/127.0.0.1`
- comment out/remove entry in `/etc/hosts`
- run `valet restart`

**`custom.loc` and any other subsequent domain are working out of the box.**

### Conclusion
Manually setting the TLD resolved the issue. If possible, investigate why it did not update automatically when changing TLD.

#### Bottom line
My case may seem a bit specific (or isolated), but it may happen to other developers who are not installing Valet+ on a fresh OSX Installation. At least, it will be documented for other developers if they run into an issue like this one. 

If you need some additional information about this issue, let me know.

Thanks!

**Comment by SanderAtom** [comment]
He @dvrgoc, thanks for the explanation of the problem and solution. Did you change the TLD with `valet domain loc` ? I see in the code that it should remove the old domain file before installing the new one (`~/.composer/vendor/weprovide/valet-plus/cli/Valet/DnsMasq.php::updateDomain()`):
```
var $resolverPath = '/etc/resolver';

[...]

$this->files->unlink($this->resolverPath.'/'.$oldDomain);
$this->install($newDomain);
```
The install also creates the `~/.valet/dnsmasq.conf` file again with the new TLD. Maybe it was unable to remove/overwrite the files due to permissions?

**Comment by XPOL555** [comment]
Another big difference for me is `dnsmasq`. Running `brew uninstall dnsmasq` and fixing last lines of `/usr/local/etc/dnsmasq.conf` file before running valet-plus's `valet install` fixed it for me.

**Comment by XPOL555** [comment]
for _fixing_ i mean the `conf-file` property that in valet 2 changed

**Comment by Neodork** [comment]
Dear Contributor,

With the release of Valet+ `2.0.0` the Valet+ development team and contributors have implemented the [contribution guidelines](https://github.com/weprovide/valet-plus/blob/master/CONTRIBUTING.md#issue-templates). The contribution guidelines aim to improve the support and workflow of Valet+. For further information why this was implemented please check https://github.com/weprovide/valet-plus/issues/393.

We're closing this ticket because it does not follow the correct format. If you feel your ticket is still relevant we encourage you to re-open your ticket using the [correct format](https://github.com/weprovide/valet-plus/issues/new/choose).

Kind regards,
The Valet+ team
